src/Tribe/Attendees.php

```diff
@@ -141,10 +141,52 @@ public function event_action_links( $event_id ) {
 	 *
 	 * @since 4.6.2
 	 */
-	public function print_checkedin_totals() {
-		$total_checked_in = Tribe__Tickets__Main::instance()->attendance_totals()->get_total_checked_in();
+	public function print_checkedin_totals( $post_id ) {
```

Comment:
Since this `print_checkedin_totals` originally had no arguments should we default $post_id to null and bail if it's null? Also looks like the `@since` needs to be updated.

Reply:
Good call @redscar !